bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -2,18 +2,27 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "mozorg/base-resp.html" %}
+{% extends "base-pebbles.html" %}
 
-{% add_lang_files "mozorg/plugincheck-redesign" %}
+{% add_lang_files "mozorg/plugincheck-redesign" "firefox/shared" %}
```

Comment:
Adding a shared file introduces a lot of complexity (you can't activate the file unless the shared one is complete). If it's only for the newsletter strings, it's better to expose them and import translations if available.